source/installation/kafka-deployments/install-confluent-kafka.txt

```diff
@@ -15,3 +15,138 @@ Install with Confluent Cloud
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+You can install Relational Migrator with Kafka using 
+`Confluent Cloud <https://docs.confluent.io/cloud/current/get-started/confluent-cloud-basics.html#ccloud>`__. 
+This deployment method provides the reliability of Kafka without having you 
+manage your own Kafka cluster. 
+
+Before you Begin
+----------------
+
+Due to restrictions by Confluent, you are only able to use Basic, Standard, and 
+Dedicated clusters. For most use cases, the basic and standard cluster are enough 
+to handle migrating large volumes of data.
+
+Steps
+-----
+
+.. procedure::
+   :style: normal 
+
+   .. step:: Create a Confluent Cloud environment
+
+      Go to `Confluent Cloud <https://www.confluent.io/>`__ and create a new environment. 
+
+   .. step:: Enable schema registry
+
+      Enable schema registry and select a region.
+
+      If you have more than 1000 tables being migrated, use the Advanced schema registry tier.
```

Comment:
[suggestion]

Consider: "If you have more than 1000 tables to migrate...".

Reply:
Updated.